Packs/URLHaus/Integrations/URLHaus/URLHaus.py

```diff
@@ -761,7 +761,10 @@ def main():
     try:
         demisto_params = demisto.params()
         command = demisto.command()
+
         auth_key = demisto_params.get("credentials", {}).get("password")
+        if not auth_key:
```

Comment:
@YuvHayun  when showing error to customer do we use wording "param" or "parameter"?

Reply:
parameter